pkg/release/create.go

```diff
@@ -136,5 +162,45 @@ func CreateRelease(name, base, releases, provider string, components, apps []str
 		return microerror.Mask(err)
 	}
 
+	// Update releases.json
+	releasesJSONPath := filepath.Join(providerDirectory, "releases.json")
+	releasesData, err := os.ReadFile(releasesJSONPath)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	var releasesJson ReleasesJsonData
+	err = json.Unmarshal(releasesData, &releasesJson)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	newReleaseInfo := ReleaseJsonInfo{
+		Version:          newVersion.String(),
+		IsDeprecated:     false,
+		ReleaseTimestamp: now.Time.Format(time.RFC3339),
+		ChangelogUrl:     fmt.Sprintf("https://github.com/giantswarm/releases/blob/master/%s/%s/README.md", provider, releaseDirectory),
+		IsStable:         true,
+	}
+
+	releasesJson.Releases = append(releasesJson.Releases, newReleaseInfo)
+
+	// sort releases in json by version
+	sort.SliceStable(releasesJson.Releases, func(i, j int) bool {
+		vi := semver.MustParse(releasesJson.Releases[i].Version)
+		vj := semver.MustParse(releasesJson.Releases[j].Version)
+		return vi.LessThan(vj)
+	})
```

Comment:
The `releases.json` file can currently be regenerated in `cd releases && go run renovate.go` AFAIK. Should we throw away one of the implementations?

(Not blocking this PR.)

Reply:
IMHO we can drop `go run renovate.go` after the we merge this PR